lib/Parser/Parser.cpp

```diff
@@ -1053,8 +1060,11 @@ bool Parser::parseLine(std::string &ErrStr) {
       Block *B = 0;
 
       if (IK == Inst::Var) {
+        // TODO: Maybe, we should initialize lower bound and upper bound to min/max value
+        // of int range w.r.t. width. for now, its initialized to 0.
         llvm::APInt Zero(InstWidth, 0, false), One(InstWidth, 0, false),
-                    ConstOne(InstWidth, 1, false);
+                    ConstOne(InstWidth, 1, false), Lower(InstWidth, 0, false),
+                    Upper(InstWidth, 0, false);
```

Comment:
why not just create a constantrange here instead of separate lower and upper variables?

Reply:
please do this